examples/llm_inference/android/app/src/main/java/com/google/mediapipe/examples/llminference/InferenceModel.kt

```diff
@@ -37,7 +37,9 @@ class InferenceModel private constructor(context: Context) {
     }
 
     fun generateResponseAsync(prompt: String) {
-        llmInference.generateResponseAsync(prompt)
+        // Add the gemma prompt prefix to trigger the response.
+        val gemmaPrompt = prompt + "<start_of_turn>model\n"
+        llmInference.generateResponseAsync(gemmaPrompt)
```

Comment:
Was this tested with Gemma2? I ran into an issue earlier with responses not ending because the model was trained with two different types of end tokens, but I don't see that adjusted for here.

Reply:
Yes